packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -38,6 +38,7 @@ export class OdspSummaryUploadManager {
         private readonly getStorageToken: InstrumentedStorageTokenFetcher,
         logger: ITelemetryLogger,
         private readonly epochTracker: EpochTracker,
+        private readonly hostPolicy: HostStoragePolicy,
```

Comment:
Maybe just pass the flag only like at other places instead of whole storagePolicy.

Reply:
Done